app/src/main/java/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -0,0 +1,306 @@
+package io.homeassistant.companion.android.sensors
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.os.Build
+import android.util.Log
+import androidx.annotation.RequiresApi
+import androidx.car.app.hardware.common.CarValue
+import androidx.car.app.hardware.info.EnergyLevel
+import androidx.car.app.hardware.info.EvStatus
+import androidx.car.app.hardware.info.Mileage
+import androidx.car.app.hardware.info.Model
+import androidx.core.content.ContextCompat
+import androidx.lifecycle.DefaultLifecycleObserver
+import io.homeassistant.companion.android.BuildConfig
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.common.sensors.SensorManager
+import io.homeassistant.companion.android.vehicle.HaCarAppService
+
+class CarSensorManager :
+    SensorManager,
+    DefaultLifecycleObserver {
+
+    companion object {
+        internal const val TAG = "CarSM"
+
+        private val fuelLevel = SensorManager.BasicSensor(
+            "car_fuel_level",
```

Comment:
Lets drop the `_level` as it does not add any value since there is already a unit and icon present. The phones battery level will remain as it has been there for a long time but for new sensors we are requested to remove it as it is not necessary. Please also update the strings to remove the word `Level` from the sensor name.